src/sqlfluff/core/templaters/jinja.py

```diff
@@ -116,7 +116,13 @@ def _extract_libraries_from_config(self, config):
             spec = importlib.util.spec_from_file_location(module_name, file_path)
             lib = importlib.util.module_from_spec(spec)
             spec.loader.exec_module(lib)
-            libraries[module_name] = lib
+            if module_name == "__init__":
+                # Allows setting root-level functions via __init__.py
```

Comment:
```suggestion
                # Allows setting root-level functions via library __init__.py
```
__init__.py is common so be clear which one we're talking about